spi/src/main/java/org/opensearch/jobscheduler/spi/schedule/CronSchedule.java

```diff
@@ -86,59 +93,71 @@ void setExecutionTime(ExecutionTime executionTime) {
         this.executionTime = executionTime;
     }
 
-    @VisibleForTesting
-    ZoneId getTimeZone() {
+    public ZoneId getTimeZone() {
         return this.timezone;
     }
 
-    @VisibleForTesting
-    String getCronExpression() {
+    public String getCronExpression() {
         return this.expression;
     }
 
+    public Long getDelay() { return this.scheduleDelay; }
+
+    @VisibleForTesting
+    public void setDelay(Long delay) { this.scheduleDelay = delay; }
```

Comment:
remove public?

Reply:
I removed the setter, but currently the public getter is still needed in [rollup.kt](https://github.com/opensearch-project/index-management/pull/147/files#diff-8aaa165b68190cf1eaab9b0c167746357d53f7b23b2ffef9046e5ba51c2cab23) as a method to check if the job schedule delay matches the rollup job delay

Reply:
I was referring to just the setter